src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,25 @@
+import { Link } from 'react-router'
+import React from 'react'
+
+const Breadcrumbs = props => {
+  const { crime, state } = props.params
+  return (
+    <ul className='breadcrumbs clearfix list-style-none px0'>
+      <li className='left mr2'>
+        <Link to='/explorer'>Explorer</Link>
```

Comment:
I don't know if all breadcrumbs will be under "explorer" views but I hardcoded it for now.